Makefile

```diff
@@ -1,25 +1,29 @@
 DOCKER ?= docker
+CONTAINER ?= container
 # Installation directories.
 PREFIX ?= $(DESTDIR)/usr
 BINDIR ?= $(PREFIX)/bin
 MANDIR ?= $(PREFIX)/share/man
 UNITDIR ?= $(PREFIX)/lib/systemd/system
-DSSLIBDIR ?= $(PREFIX)/lib/docker-storage-setup
+CSSLIBDIR ?= $(PREFIX)/lib/container-storage-setup
 SYSCONFDIR ?= $(DESTDIR)/etc/sysconfig
 
 .PHONY: clean
 clean:
 	-rm -rf *~ \#* .#*
 
 .PHONY: install
-install:
-
-	install -D -m 755 ${DOCKER}-storage-setup.sh ${BINDIR}/${DOCKER}-storage-setup
+install-docker:
 	install -D -m 644 ${DOCKER}-storage-setup.service ${UNITDIR}/${DOCKER}-storage-setup.service
```

Comment:
No, I don't believe so.  I believe @lsm5 is either not using the Makefile or is overriding the docker.service file elsewhere.  If he is using it he will need to move it docker.spec.